src/components/MapsPanel/MapsPanel.tsx

```diff
@@ -40,12 +40,12 @@ export default function MapsPanel(): JSX.Element {
     orientation,
   });
 
```

Comment:
you might have gotten it from my PR, the default map is set to Polus instead of Skeld (cant select the line since it wasn't changed)

Reply:
Yeah I did not change anything from that, I can do so if you want.

Reply:
it doesn't matter. you can leave it as is. just pointing it out.

unrelated, i saw someone left a feedback and said their language is defaulted to german? but im pretty sure we didnt touch any of the i18n stuff

Reply:
Yeah idk how or why that happens, never seen it before. I have it default to en-US.